tests/perf/test_anomaly.py

```diff
@@ -5,32 +5,40 @@
 
 from __future__ import annotations
 
+import logging
 from pathlib import Path
+from typing import ClassVar
 
 import pytest
 
 from .benchmark import Benchmark
 from .conftest import PerfTestBase
 
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
```

Comment:
It seems that default value of the deterministic is consider as `False`. Then, how about using this code at `tests/perf/conftest.py::fxt_deterministic` and using that fixture? If then, we don't have to override it in every tasks.

Reply:
No. default deterministic value is None. If user does not set deterministic, it will be used the task specific fixed value.

Reply:
Each task has the default deterministic value differently. e.g. true for action, classification and detection. false for anomaly, ins seg, sseg and vp.